### ISSUE TITLE
Explain the use cases for your module

### ISSUE THREAD

**Issue by hitautodestruct** [sterlingw/SimpleSpy.js#4]
Readme should explain the use cases for your module or at least give a reference to some link.

**Comment by sterlingw** [comment]
The readme clearly illustrates that the purpose of `espionage` is to create spies. Do you have a specific question about the module?

**Comment by hitautodestruct** [comment]
Of course. But what would you use spies for?

**Comment by tom2strobl** [comment]
I agree with @hitautodestruct, there should at least be a link to an external reference of what a spy is. Maybe there's a good Jasmine reference?

**Comment by sterlingw** [comment]
@hitautodestruct @tom2strobl I see. I'll put something in the README explaining the concept of test spies. Thanks!
